submodules/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ include("jsoncpp.cmake")
 if(VRPN_USE_LOCAL_JSONCPP)
 	set(JSONCPP_FOUND ${JSONCPP_FOUND} PARENT_SCOPE)
 	set(JSONCPP_INCLUDE_DIRS ${JSONCPP_INCLUDE_DIRS} PARENT_SCOPE)
-	set(JSONCPP_LIBRARIES ${JSONCPP_LIBRARIES} PARENT_SCOPE)
-	set(JSONCPP_SOURCES ${JSONCPP_SOURCES} PARENT_SCOPE)
+	project(jsoncpp_lib)
```

Comment:
Is it OK to just put a `project()` call in there like that? What are the advantages of putting that call in vs. leaving it out (what I'd normally do, and what I did with HIDAPI elsewhere: https://github.com/rpavlik/HDK-Logger/blob/master/vendor/hidapi.cmake )

Also, this does actually change the behavior - it no longer statically includes jsoncpp in the same archive/lib as VRPN, thus making another file that should be installed when building as a static library. (Because it doesn't specify the library type, actually, it would also build a dll instead if CMake were switched to that mode, which is probably also unintentionally.) The alternative would be to make an "object" target (I think?) but that requires a fairly new CMake, not sure how new offhand, and at least for now I've been trying to keep the cmake requirements relatively old for VRPN because it's used on a lot of "outdated" systems.